src/Renderer/Camera.js

```diff
@@ -13,6 +14,7 @@ function Camera(crs, width, height, options = {}) {
     this._viewMatrix = new THREE.Matrix4();
     this.width = width;
     this.height = height;
+    this.handleCollision = false;
```

Comment:
For now I'd prefer this option to exist only on the controls that support it.
So I'd add a `handleCollision` to `GlobeControls` and `PlanarControls` and some code in `PlanerView`/`GlobeView` constructor. For instance:
```
this.controls.handleCollision = options.handleCollision || false;
```